core/src/test/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeCommandBuilderSpec.scala

```diff
@@ -0,0 +1,64 @@
+package com.databricks.labs.remorph.parsers.snowflake
+
+import com.databricks.labs.remorph.parsers.intermediate._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.mockito.MockitoSugar
+
+class SnowflakeCommandBuilderSpec
+    extends AnyWordSpec
+    with SnowflakeParserTestCommon
+    with Matchers
+    with MockitoSugar
+    with IRHelpers {
+
+  override protected def astBuilder: SnowflakeCommandBuilder =
+    new SnowflakeCommandBuilder
+
+  "translate Declare to CreateVariable Expression" in {
+    example(
+      "x number default 0;",
```

Comment:
can you use UPPERCASE for keywords everywhere in tests, so that it's easier for reviewers to understand